templates/ClientOptionsFile.stencil

```diff
@@ -28,5 +30,8 @@ public struct {{ model.name }}Options: AzureClientOptions {
         self.logger = logger
         self.telemetryOptions = telemetryOptions
         self.transportOptions = transportOptions ?? TransportOptions()
+
+        // TODO: To be implemented
+        self.cancellationToken = nil
```

Comment:
Is this a follow-up item, or do you plan to put this into this PR? The AzureCore PR has been merged.

Reply:
a Follow up item. Not part of this pr.

Reply:
Fixed.